python/console/console_output.py

```diff
@@ -85,6 +81,36 @@ def isatty(self):
         return False
 
 
+FULL_HELP_TEXT = QCoreApplication.translate("PythonConsole", """QGIS Python Console
+======================================
+
+The console is a Python interpreter that allows you to execute python commands.
+Modules from QGIS (analysis, core, gui, 3d) and Qt (QtCore, QtGui, QtNetwork,
+QtWidgets, QtXml) as well as Python's math, os, re and sys modules are already
+imported and can be used directly.
+
+Useful variables:
+
+- iface.mainWindow() will return the Qt Main Window
```

Comment:
```suggestion
- iface will return the current QGIS interface, class 'QgisInterface'
- iface.mainWindow() will return the Qt Main Window
```

I guess it's handy to have `iface` which leads to all variables, and also the class name
Feel free to rephrase.